account_asset_management/models/account_asset.py

```diff
@@ -495,15 +474,16 @@ def name_search(self, name, args=None, operator="ilike", limit=100):
             if operator in expression.NEGATIVE_TERM_OPERATORS:
                 domain = ["&", "!"] + domain[1:]
         assets = self.search(domain + args, limit=limit)
-        return assets.name_get()
+        return assets._compute_display_name()
```

Comment:
This is not correct. You can't call a compute method in a search.

Reply:
Instead of overriding `name_search`, use `_rec_names_search`, as stated in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0 (it wasn't changed in previous migration)

Reply:
@pedrobaeza done